docs/tutorials/Canopy/soil_canopy_tutorial.jl

```diff
@@ -109,6 +110,67 @@ include(
 # Define the parameters for the soil model and provide them to the model 
 # parameters struct:
 
+# Heterotrophic respiration parameters
+P_sfc = FT(101e3)
+ν = FT(0.556)
```

Comment:
how do we ensure this is the same as the soil porosity? we should only define 1x

Reply:
I just did `v = soil_v` - which is what I had in experiment, I forgot to edit it for the tutorial. 
Do you think that's OK?

Reply:
I think it is OK for now! Once we go to global runs, soil porosity will be stored in the aux state because it will vary everywhere. so we'll have to make sure this model gets the right value for each location.